quizapp/src/main/java/billennium/quizapp/service/QuizService.java

```diff
@@ -150,4 +150,12 @@ private void setTimeToResolve(Long quizExecutedId, Question question) {
     private boolean checkDate(LocalDateTime now, LocalDateTime timeLimit) {
         return now.isBefore(timeLimit) || now.isEqual(timeLimit);
     }
+
+    public void stopQuiz(QuizEndDto quizEndDto) {
+        Optional<QuizExecuted> quizToStop = quizExecutedRepository.findById(quizEndDto.getQuizId());
```

Comment:
czy tak to musi działać ? nie można odkryć w kodzie że jeżeli jest wyslana i zapisywana odpowiedz na ostatnie pytanie to wtedy robimy DONE ?

Reply:
ta metoda jest uzywana  w chwili gdy podczas testu np na 3z  5 pytan wylaczysz quiz